rules/windows/process_access/proc_access_win_direct_syscall_ntopenprocess.yml

```diff
@@ -54,7 +54,9 @@ detection:
         TargetImage: 'C:\Program Files (x86)\CCleaner Browser\Application\CCleanerBrowser.exe'
         SourceImage: 'C:\Program Files (x86)\CCleaner Browser\Application\CCleanerBrowser.exe'
     falsepositives13:
-        TargetImage|endswith: '\Discord.exe'
+        TargetImage|endswith:
+            - '\Discord.exe'
+            - 'C:\WINDOWS\system32\AUDIODG.EXE'
```

Comment:
Do you have the `SourceImage` ? Since it's a high rule. It's better to tighten it as best we can.